python-interface/src/MiniBotScripts/Buddybot.py

```diff
@@ -0,0 +1,190 @@
+import Note_Library as nl
+import live_audio_sample as las
+import random
+
+#Used for individual emotion testing 
+TEST_EMOTION = 0; #TODO remove once emotions finalized
```

Comment:
The individual reactions, no. Should I remove the testing variable anyway? It makes it easier to test individual ones while keeping the final structure I was hoping for.